+tests/test_base_functionality.m

```diff
@@ -0,0 +1,33 @@
+classdef test_base_functionality < matlab.unittest.TestCase
+
+    properties
+        tol = 1e-4;
+    end
+    
+    methods(Test)
+        % Test methods
+        
+        function test_run(self)
+            example_run;
+            assert(abs(n.t - 0.025) < self.tol);
+        end
+        
+        function test_vector(self)
+            % Example to run.
+            example_vector;
+            % Asserts for first vector.
+            assert(isa(v, "neuron.Vector"));
+            assert(abs(length(v) - 11) < self.tol);
```

Comment:
With testcases, would mostly want to use
self.verifyXXX and where needed self.assertXXX, instead of assert(), where there are dedicated qualifications for various cases, as per https://nl.mathworks.com/help/matlab/matlab_prog/types-of-qualifications.html

However, i'd say it can be a follow-up issue to make those changes.